static_src/constants.js

```diff
@@ -22,6 +22,8 @@ const entityHealth = keymirror({
   warning: null,
   // Something is definitely not right and requires some attention
   error: null,
+  // Entity process was successful
+  finish: null,
```

Comment:
leave it as-is, we should revisit it later though along with cg-stlye. because finish != success in all cases.